recipes/openimageio/all/conanfile.py

```diff
@@ -63,260 +64,248 @@ class OpenImageIOConan(ConanFile):
         "with_openvdb": False,  # FIXME: broken on M1
         "with_ptex": True,
         "with_libwebp": True,
+        "build_testing": False,
     }
 
-    short_paths = True
-    generators = "cmake", "cmake_find_package"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
 
     def requirements(self):
         # Required libraries
-        self.requires("zlib/1.2.13")
-        self.requires("boost/1.78.0")
-        self.requires("libtiff/4.5.1")
-        self.requires("openexr/2.5.7")
+        self.requires("zlib/[>=1.2.11 <2]")
+        self.requires("boost/1.84.0")
+        self.requires("libtiff/4.6.0")
+        self.requires("imath/3.1.9", transitive_headers=True)
+        self.requires("openexr/3.2.1")
         if self.options.with_libjpeg == "libjpeg":
             self.requires("libjpeg/9e")
         elif self.options.with_libjpeg == "libjpeg-turbo":
-            self.requires("libjpeg-turbo/2.1.5")
-        self.requires("pugixml/1.12.1")
+            self.requires("libjpeg-turbo/3.0.2")
+        self.requires("pugixml/1.14")
         self.requires("libsquish/1.15")
-        self.requires("tsl-robin-map/1.0.1")
-        self.requires("fmt/8.1.1")
+        self.requires("tsl-robin-map/1.2.1")
+        if Version(self.version) >= "2.4.17.0":
+            self.requires("fmt/10.2.1", transitive_headers=True)
+        else:
+            self.requires("fmt/9.1.0", transitive_headers=True)
 
         # Optional libraries
         if self.options.with_libpng:
-            self.requires("libpng/1.6.40")
+            self.requires("libpng/1.6.42")
```

Comment:
No luck, now there  is a conflict on libpng releases because of freetype.
Perhaps we could relax the requirement on libpng:
```suggestion
            self.requires("libpng/[>=1.6.40 <2]")
```

Reply:
Not sure it is is only one cause there is also complaints about libtiff (which I didn't touch) https://c3i.jfrog.io/c3i/misc/logs/pr/19950/47-linux-gcc11/openimageio/2.4.7.1//543ed657bef9ccadbcb98df436e25612de3b2d3f-build.txt

So I don't really get how one is supposed to work with this. Just pushing things and waiting for hours in the CI and then push another try for dependencies can't be the solution?!

(To be honest I don't get the prebuild packages concept anyway as I always run into mismatched dependencies and stuff. Also for some stuff I just sometimes anyway want to configure options so prebuild hasn't worked for me for all but quick tests.)